diff --git a/src/util/config/config.h b/src/util/config/config.h
index f947272..537d0ef 100644
--- a/src/util/config/config.h
+++ b/src/util/config/config.h
@@ -2,7 +2,8 @@
 
 #include <string>
 #include <unordered_map>
-
+#include <utility>
+#include <cstdint>
 namespace dxvk {
 
   /**
diff --git a/src/util/util_bit.h b/src/util/util_bit.h
index 7682ace..e7286aa 100644
--- a/src/util/util_bit.h
+++ b/src/util/util_bit.h
@@ -1,5 +1,5 @@
 #pragma once
-
+#include <cstdint>
 #ifndef _MSC_VER
 #if defined(__WINE__) && defined(__clang__)
 #pragma push_macro("_WIN32")
diff --git a/src/util/util_matrix.cpp b/src/util/util_matrix.cpp
index 2c5e931..df9a3c4 100644
--- a/src/util/util_matrix.cpp
+++ b/src/util/util_matrix.cpp
@@ -1,5 +1,5 @@
 #include "util_matrix.h"
-
+#include <cstdint>
 namespace dxvk {
 
         Vector4& Matrix4::operator[](size_t index)       { return data[index]; }